background/services/provider-bridge/db.ts

```diff
@@ -0,0 +1,79 @@
+import Dexie from "dexie"
+
+/**
+ * An object describing a set of permissions granted to a given provider.
+ * Generally tracked by dApp as part of DAppPermissions.
```

Comment:
```suggestion
 * Generally tracked by dAppID as part of DAppPermissions.
```

Pretty sure I messed this one up in my PR haha.